tests/backstop_data/engine_scripts/puppet/civicase/find.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = async (engine, scenario, vp) => {
+  await Promise.all([
```

Comment:
you sure we need this? Have you tried without?

Reply:
Yes. We have to use this. It randomly timeout if we don't follow this logic.